cmd/descheduler/app/descheduler.go

```diff
@@ -231,6 +232,8 @@ func healthzHandler(w http.ResponseWriter, _ *http.Request) {
 }
 
 func metricsHandler() http.Handler {
+	ctrlmetrics.Registry.MustRegister(versionmetrics.NewBuildInfoCollector())
```

Comment:
Can we register the metric outside of the handler function? It seems the handler is a highly cohesive function and might be involked every where.